### PR TITLE
docs(plugin): refactor request-validation.md

### PR DESCRIPTION
**Pull request by lunarwhite**
### Description

update documents under `docs/zh/latest/plugins/` along with `docs/en/latest/plugins/`

### Checklist

- [x] I have explained the need for this PR and the problem it solves
- [x] I have explained the changes or the new features added to this PR
- [ ] I have added tests corresponding to this change
- [x] I have updated the documentation to reflect this change
- [x] I have verified that this change is backward compatible (If not, please discuss on the [APISIX mailing list](https://github.com/apache/apisix/tree/master#community) first)

### CODE REVIEW

File: docs/zh/latest/plugins/request-validation.md [hunk 1]
```diff
@@ -23,27 +28,30 @@ title: request-validation
...
-| rejected_code | integer | 可选        |   400      |    [200,...,599]   | 自定义拒绝状态码 |
-| rejected_msg | string | 可选        |         |       | 自定义拒绝信息 |
+| header_schema    | object | 否        |         |       | `header` 数据的 `schema` 数据结构。 |
+| body_schema      | object | 否        |         |       | `body` 数据的 `schema` 数据结构。   |
+| rejected_code | integer | 否        | 400      | [200,...,599]   | 当请求被拒绝时要返回的状态码。 |
+| rejected_msg | string | 否        |         |       | 当请求被拒绝时返回的信息。 |
+
+:::note 注意
+
+`header_schema` 与 `body_schema` 至少填写其中一个。
```
Comment (SylviaBABY) [inline docs/zh/latest/plugins/request-validation.md:44]:
```suggestion
需保证 `header_schema` 与 `body_schema` 配置项中，至少填写其中一个。
```

File: docs/zh/latest/plugins/request-validation.md [hunk 2]
```diff
@@ -23,27 +28,30 @@ title: request-validation
...
-| rejected_code | integer | 可选        |   400      |    [200,...,599]   | 自定义拒绝状态码 |
-| rejected_msg | string | 可选        |         |       | 自定义拒绝信息 |
+| header_schema    | object | 否        |         |       | `header` 数据的 `schema` 数据结构。 |
+| body_schema      | object | 否        |         |       | `body` 数据的 `schema` 数据结构。   |
+| rejected_code | integer | 否        | 400      | [200,...,599]   | 当请求被拒绝时要返回的状态码。 |
+| rejected_msg | string | 否        |         |       | 当请求被拒绝时返回的信息。 |
+
+:::note 注意
+
+需保证 `header_schema` 与 `body_schema` 配置项中，至少填写其中一个。
```
Comment (hf400159) [inline docs/zh/latest/plugins/request-validation.md:44]:
```suggestion
启用该插件时，至少需要配置 `header_schema` 和 `body_schema` 属性中的任意一个，两者也可以同时使用。
```

File: docs/zh/latest/plugins/request-validation.md [hunk 3]
```diff
@@ -68,39 +76,9 @@ curl http://127.0.0.1:9080/apisix/admin/routes/5 -H 'X-API-KEY: edd1c9f034335f13
...
-        "type": "roundrobin",
-        "nodes": {
-            "127.0.0.1:8080": 1
-        }
-    }
-}'
-```
-
-## 示例
+以下示例展示了不同的验证场景下的 `request-validation` 插件 JSON 配置：
```
Comment (hf400159) [inline docs/zh/latest/plugins/request-validation.md:79]:
```suggestion
以下示例展示了不同验证场景下该插件的 JSON 配置：
```

**Review by hf400159 — COMMENTED** [review]
LGTM 👍

**Review by SylviaBABY — APPROVED** [review]

**Review by SkyeYoung — APPROVED** [review]

**Review by bzp2010 — APPROVED** [review]

**Pull request merged**